### ISSUE TITLE
Rest endpoints `/utils/*` accept GET parameters with oversized ErgoTree/Address

### ISSUE THREAD

**Issue by pragmaxim** [ergoplatform/ergo#1868]
`ErgoTree/Address` can be quite huge and violate HTTP limit for 2000 characters long URI ... They should be part of `POST` body.

**Comment by jellymlg** [comment]
> `ErgoTree/Address` can be quite huge and violate HTTP limit for 2000 characters long URI ... They should be part of `POST` body.

good point, should I change the enpoints in #1859 ?

**Comment by aslesarenko** [comment]
> is there a limit for ErgoTree size?
Yes, size of a Box is 4K bytes, which includes the ErgoTree and other registers.

**Comment by abebeos** [comment]
@aslesarenko , it looks #1871 "Closes" did not function. Issue should be closable.

**Comment by abebeos** [comment]
@kushti , closable?

**Comment by Ergologica** [comment]
Checking before anyone spends time on this: it looks mostly done already.

`ErgoUtilsApiRoute` on master has POST variants for both routes that take an ErgoTree or an address:

- `POST /utils/address` — `validateAddressPostR`
- `POST /utils/ergoTreeToAddress` — `ergoTreeToAddressPostR`

Both are in `openapi.yaml` too (`/utils/address`, `/utils/ergoTreeToAddress`), alongside the GET forms. `/utils/hash/blake2b` was POST-only from the start.

What is left is `GET /utils/addressToRaw/{address}`, which has no POST form. A P2S address encodes a whole ErgoTree, so that one can indeed go past the 2000 character limit. The other remaining GET, `rawToAddress/{pubKeyHex}`, takes a 33-byte group element — 66 hex characters — so it is never affected.

So the question is whether the `addressToRaw` variant alone is still wanted, and whether the bounty still applies to that remainder. I am happy to send a small PR (POST route, openapi entry, a case in `UtilsApiRouteSpec`) if it is; otherwise this could probably be closed.
